star_tides/services/databases/mongo/schemas/guidance_schema.py

```diff
@@ -0,0 +1,43 @@
+''' star_tides.services.databases.mongo.schemas.guidance_schema
+'''
+from star_tides.services.databases.mongo.models.guide_model import Guide
+from star_tides.services.databases.mongo.schemas.location_schema import (
+    LocationSchema
+)
+from marshmallow import Schema, fields, validates, post_load
+
+
+class GuidanceSchema(Schema):
+    ''' Schema for guidance
+    '''
+    id = fields.String(attribute='id', dump_only=True)
+    author = fields.String(required=True)
```

Comment:
I was thinking that a guide has a single author. I can give you a better sense of what the Guide object should look like in a frontend PR.